src/roll-chromium.ts

```diff
@@ -155,35 +147,86 @@ export async function rollChromium(
   return forkRef.substr(11);
 }
 
+function prText(previousChromiumVersion: string, chromiumVersion: string, branchName: string) {
+  const isLKGR = !chromiumVersion.includes('.');
+  const shortVersion = isLKGR ? chromiumVersion.substr(11) : chromiumVersion;
+  const shortPreviousVersion = isLKGR ? previousChromiumVersion.substr(11) : previousChromiumVersion;
+  const diffLink = `https://chromium.googlesource.com/chromium/src/+/${previousChromiumVersion}..${chromiumVersion}`;
+  return {
+    title: `chore: bump chromium to ${shortVersion} (${branchName})`,
+    body: `Updating Chromium to ${shortVersion}${isLKGR ? ' (lkgr)' : ''}.
+
+See [all changes in ${shortPreviousVersion}..${shortVersion}](${diffLink})
+
+<!--
+Original-Chromium-Version: ${previousChromiumVersion}
+-->
+
+Notes: ${isLKGR ? 'no-notes' : `Updated Chromium to ${chromiumVersion}.`}`,
+  };
+}
+
 export async function rollChromium4(
-  electronBranch: string,
+  electronBranch: {name: string, commit: {sha: string}},
   chromiumVersion: string,
-): Promise<string | null> {
-  d(`triggered for electronBranch=${electronBranch} chromiumVersion=${chromiumVersion}`);
+): Promise<void> {
+  d(`roll triggered triggered for electronBranch=${electronBranch.name} chromiumVersion=${chromiumVersion}`);
   const github = await getOctokit();
-  // Get current SHA of {electronBranch} on electron/electron
-  const electronReference = await github.git.getRef({
+
+  // Look for a pre-existing PR that targets this branch to see if we can update that.
+  const existingPrsForBranch = await github.pulls.list({
+    per_page: 100, // TODO: paginate
+    base: electronBranch.name,
     owner: 'electron',
     repo: 'electron',
-    ref: `heads/${electronBranch}`,
+    state: 'open',
   });
-  const electronSha = electronReference.data.object.sha;
-  const forkRef = `refs/heads/roller/chromium-${chromiumVersion}-${Date.now()}`;
+  const myPrs = existingPrsForBranch.data.filter((pr) => pr.user.login === PR_USER);
+
+  if (myPrs.length) {
```

Comment:
We should add the equivalent of `FATAL()` if `myPrs.length > 2`.  There is no reasonable use case for that so we should log a giant error here and only update the first one we find.

Reply:
Logs for this are not watched with any regularity. I agree that there's no reason for there to be multiple PRs open here but I don't think there's any harm in updating them all?